src/modules/data/validator/index.ts

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+
+    The class that defines the Validator.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+import Ajv from 'ajv';
+export { IBitField } from '../types/BitField';
+export { IBlock } from '../types/Block';
+export { IBlockHeader } from '../types/BlockHeader';
+export { IEnrollment } from '../types/Enrollment';
+export { IHeight } from '../types/Height';
+export { ITransaction } from '../types/Transaction';
+export { ITxInputs } from '../types/TxInputs';
+export { ITxOutputs } from '../types/TxOutputs';
+
+const ajv = new Ajv();
+
+/**
+ * Generic function of validation
+ */
+export interface ValidateFunction<T> extends Ajv.ValidateFunction {
+    _t?: T; // stop linter from complaining about unused T variable
+}
+
+/**
+ * Class for validating JSON data
+ */
+export class Validator
+{
+    /**
+     * The map of validation functions created to reuse
+     * a once created validation function.
```

Comment:
```suggestion
     * An once created validation function.
```

Reply:
```
The map of validation functions created to reuse -
an once created validation function.
```
I'll change it like this. Do you like it?